lib/spack/spack/spec.py

```diff
@@ -186,11 +186,11 @@ class InstallStatus(enum.Enum):
     Options are artificially disjoint for display purposes
     """
 
-    installed = "@g{[+]}  "
-    upstream = "@g{[^]}  "
-    external = "@g{[e]}  "
-    absent = "@K{ - }  "
-    missing = "@r{[-]}  "
+    installed = "@g{[+]}"
+    upstream = "@g{[^]}"
+    external = "@g{[e]}"
+    absent = "@K{ - }"
+    missing = "@r{[-]}"
```

Comment:
Can we make these uppercase, like other enums in the code?

Reply:
done